src/components/layoutComponents/header/header.js

```diff
@@ -23,7 +23,7 @@ const Header = () => {
               {data.site.siteMetadata.title}
         </Navbar.Brand>
         <Nav className="mr-auto">
-          <Nav.Link as={Link} to='/'>
+          <Nav.Link as={Link} to='/' activeClassName={headerStyles.activeNavItem}>
```

Comment:
Maybe just `activeStyle={{ color: "#ffffff" }}`?

Reply:
Yes, I tried to fix the problem and forget to remove it

Reply:
`activeStyle={{ color: "#ffffff" }}` for all elements `Nav.Link` work, i checked